### ISSUE TITLE
evaluate result is too low

### ISSUE THREAD

**Issue by LordonCN** [abhi1kumar/DEVIANT#12]
Hello, thanks for your great work~
I trained for 20 epoch on 2*GTX1080, but I find evaluate result is too low, pls help.

Here is the comparison between yours(left) and mine(right):
<img width="1130" alt="Screen Shot 2023-08-03 at 10 17 14" src="https://github.com/abhi1kumar/DEVIANT/assets/46736284/5e2710fe-a932-45f1-8213-755fba58681e">

Here is my training log:
[20230802_171210.txt](https://github.com/abhi1kumar/DEVIANT/files/12246241/20230802_171210.txt)

**Comment by abhi1kumar** [comment]
Hi @LordonCN 
Thank you for showing interest in DEVIANT. You use a small batch size of **2**. We use a batch size of **12** in our [config](https://github.com/abhi1kumar/DEVIANT/blob/main/experiments/run_221.yaml#L10).

Small batch size affects the batch normalization layers in the backbones.  See [group norm](https://arxiv.org/pdf/1803.08494.pdf) paper.

There are couple of things you could try:
1. Train with a bigger batch size on a bigger GPU
2. Change the [sesn_scales](https://github.com/abhi1kumar/DEVIANT/blob/main/experiments/run_221.yaml#L28) in the config from `[0.83, 0.9, 1.0]` to `[0.83, 1.0]`. (The performance decrease is only a little)
Using two sesn_scales instead of three should reduce your memory usage, and allow 1.5 times bigger batch size.

**Comment by LordonCN** [comment]
Thanks a lot, I will try again

**Comment by LordonCN** [comment]
I just checked that the NMS threshold and the evaluate results, the results content under result_kitti/data are all empty.

I am testing to change NMS threshold to 0.05 to have a look.

**Comment by abhi1kumar** [comment]
> I just checked that the NMS threshold and the evaluate results, the results content under result_kitti/data are all empty.

Try running the inference first with our KITTI pretrained model and see if you can reproduce our results on your end.

**Comment by LordonCN** [comment]
> > I just checked that the NMS threshold and the evaluate results, the results content under result_kitti/data are all empty.
> 
> Try running the inference first with our KITTI pretrained model and see if you can reproduce our results on your end.

Yesterday I finished this step and it worked very well.
![image](https://github.com/abhi1kumar/DEVIANT/assets/46736284/e894cca8-7058-4911-a75b-68c225f8d589)

**Comment by LordonCN** [comment]
> I just checked that the NMS threshold and the evaluate results, the results content under result_kitti/data are all empty.
> 
> I am testing to change NMS threshold to 0.05 to have a look.

Summarize:
 - 20 epoch is not enough, change nms threshold to 0.01 can get predict result.
 - Train with a bigger batch size.
